### PR TITLE
Updated man page for rhn.conf file to include information on setting Satellite's SMTP server

### PR DESCRIPTION
**Pull request by csheremeta**
Added variable for SMTP server to rhn.conf file and updated the man page

### CODE REVIEW

File: spacewalk/config/usr/share/man/man5/rhn.conf.5 [hunk 1]
```diff
@@ -235,6 +235,13 @@ When importing packages, skip non-essential data that can make the import faster
 .B Default:
 0
 
+.TP
+.B "java.smtp_server"
+This parameter defines the server Satellite will use as its SMTP server.
+.IP
+.B Default:
+none
```
Comment (tchellomello) [inline spacewalk/config/usr/share/man/man5/rhn.conf.5]: @csheremeta I think you should add the default as a localhost as we can see here on the code. 

```bash
↪ grep -R WEB_SMTP * 
common/conf/ConfigDefaults.java:    public static final String WEB_SMTP_SERVER = "java.smtp_server";
common/messaging/SmtpMail.java:        smtpHost = c.getString(ConfigDefaults.WEB_SMTP_SERVER, "localhost");
```

**Comment by tlestach** [comment]
More info: https://bugzilla.redhat.com/show_bug.cgi?id=1528852

**Comment by csheremeta** [comment]
updated Default in rhn.conf man page to localhost instead of none

**Comment by csheremeta** [comment]
removed edits to rhn.conf file per comment from @tlestach on https://bugzilla.redhat.com/show_bug.cgi?id=1528852

**Pull request merged**